addons/hr_expense/models/hr_expense.py

```diff
@@ -195,8 +195,8 @@ def _convert_to_tax_base_line_dict(self, base_line=None, currency=None, price_un
             currency=currency or self.currency_id,
             product=self.product_id,
             taxes=self.tax_ids,
-            price_unit=price_unit or self.total_amount_company,
-            quantity=quantity or 1,
+            price_unit=price_unit or 0,
```

Comment:
```suggestion
            price_unit=price_unit or self.total_amount,
```
What do you think?

Reply:
@alialfie Bad Idea, it should be in the company currency (if we use a fallback) and it's used to compute the total_amounts anyway so I think setting it every time is a better solution and just default to 0 if not